requirements.txt

```diff
@@ -1,12 +1,12 @@
 #### ESSENTIAL LIBRARIES
 
-snorkel==0.9.5
+snorkel==0.9.9
```

Comment:
Do we want to test that the tutorials all still work with upgrading to the latest snorkel?